layer/manifest/XrApiLayer_dive.json

```diff
@@ -0,0 +1,14 @@
+{
+    "file_format_version": "1.0.0",
+    "api_layer": {
+        "name": "XR_APILAYER_dive",
+        "library_path": "/data/local/tmp/libXrApiLayer_dive.so",
```

Comment:
out of curiosity, how do we copy this .so file? user needs to do the copy manually or we have a script somewhere to do the copy?